### ISSUE TITLE
I'd like to join, but can't use Coinbase Pro.

### ISSUE THREAD

**Issue by mirimir** [OrchidTechnologies/orchid#15]
The problem is that Coinbase Pro won't let me create an account.

I suppose that I'll just need to wait until other exchanges start trading Bitcoin for OXT .

But it'd be great if there were an option to be updated about that through email.

**Comment by chrishasl** [comment]
Check out some [other markets on CMC](https://coinmarketcap.com/currencies/orchid/markets/)

**Comment by trees-smoke** [comment]
From what I see on coinbase pro they only have the OXT/USD pair for now, so would have to make two trades anyway if you're starting with BTC.

Before it was even listed on coinbase I went to https://1inch.exchange to get some OXT at ~$0.50/OXT.  You can also use https://uniswap.exchange/  I did the exchange using USDC, so you could sell BTC for USDC, or ETH, etc. then swap, with uniswap it exchanges to ETH first anyway, your choice.

With https://uniswapdex.com/ you can see that the uniswap market has about $12k of OXT liquidity right now.

**Comment by mirimir** [comment]
This is confusing and frustrating.

I don't use smartphones or Chrome, so I ended up creating a MEW keystore wallet. And I swapped some BTC for ETH. But it seems that neither https://1inch.exchange/ nor https://uniswap.exchange/ can connect with MEW keystore wallets. 

It'd be really cool if someone could just explain how to get from BTC to OXT. Without requiring a smartphone. If necessary, I could use Chrome.
